lib/hackney/income/use_case_factory.rb

```
@@ -110,6 +116,13 @@ def events_gateway
         Hackney::Income::SqlEventsGateway.new
       end
 
+      def search_tenancies_gateway
+        Hackney::Income::SearchTenanciesGateway.new(
+          api_host: ENV['INCOME_COLLECTION_API_HOST'],
```

`ENV` as an array has a habit of failing silently at runtime. A better method is to use fetch `ENV.fetch('INCOME_COLLECTION_API_HOST')` will fail to boot the app (and tell you what's missing)